### ISSUE TITLE
linting

### ISSUE THREAD

**Issue by jmcook1186** [jmcook1186/BioSNICAR_GO_PY#24]
pylint reveals > 100 violations of PEP8 standards in driver script alone. These are inconsequential for the correct functioning of the code but also do not conform to best practice. We should go through the linter issues and fix them. They are mostly very small issues like additional/missing whitespace after commas etc

To fix, in terminal:

in bioSNICAR environment:
`pip install pylint`

then
`pylint ./src/snicar_driver.py`

and fix issues reported in terminal

**Comment by jmcook1186** [comment]
NOTE
main outstanding linting issue is the excessive number of local variables. This can easily be resolved by passing inputs.variable as function arguments instead of unpacking inputs into local variables in each script. This is a bit laborious but is easy to do. Affects snicar_feeder.py, adding_doubling_solver.py and toon_solver.py.

e.g. 
snicar_feeder.py
```python
# DELETE LINES 34 - 58 THAT LOOK LIKE THIS
    dir_base = inputs.dir_base
    toon = inputs.toon
    add_double = inputs.add_double
    nbr_lyr = inputs.nbr_lyr
    nbr_aer = inputs.nbr_aer
    layer_type = inputs.layer_type
...
```
and then where those vars are used, replace with inputs.var, e.g.

```python
# replace dir_base with inputs.dir_base
    dir_spherical_ice_files = str(inputs.dir_base + 'Data/OP_data/480band/ice_spherical_grains/')
    dir_hexagonal_ice_files = str(inputs.dir_base + 'Data/OP_data/480band/ice_hexagonal_columns/')


```

**Comment by jmcook1186** [comment]
snicar_driver, snicar_feeder and experiments/driver now conform to pep8. 

Linting of the rest of the repo is still desired.

**Comment by tgoelles** [comment]
I would simply include black for all the formatting https://black.readthedocs.io/en/stable/. Then there is no discussion on how to format code. This can also easily be automated either via the IDE, pre-commit hooks or github actions.
If the whole project uses black, then it is also easier to check PRs because you only see the changes which are relevant and not formatting.

**Comment by jmcook1186** [comment]
Great! Thanks for the suggestion -  I will look into this next week.

**Comment by jmcook1186** [comment]
Closed via PR #34 
project adopting "black" format as suggested by T Goelles. Repo can now be automatically formatted.
